strapi/strapi
# [Docs] Set build onbrokenlinks to throw

### What does it do?

Throws errors when broken links are found rather than warning

### Why is it needed?

To prevent broken links from making it into the docs

### How to test it?

Try adding a broken link and then `yarn build` in /docs and it should throw an error.

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/pull/17114
